recipe/meta.yaml

```
@@ -9,35 +8,49 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: cfdb349d46ed3873ece4586b96aa75258726e2fa8ec21d6f00a591d98806c2f4
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: True  # [py<35]
```

https://github.com/avian2/unidecode/blob/master/setup.py#L23}

Python 3.7 or above
```suggestion
  skip: True  # [py<37]
```

---

I went based off [this](https://github.com/avian2/unidecode/blob/unidecode-1.3.8/setup.py#L23) since I thought we were supposed to be using the info related to the tagged version as opposed to what's on master.

---

I'd be curious about this also!

---

> I went based off [this](https://github.com/avian2/unidecode/blob/unidecode-1.3.8/setup.py#L23) since I thought we were supposed to be using the info related to the tagged version as opposed to what's on master.

agreed.